src/main/java/controllers/ServletController.java

```diff
@@ -0,0 +1,93 @@
+package controllers;
+
+import calculator.CalculatorService;
+import calculator.WebCalculatorFactory;
+import responseCodes.codes.StatusCode;
+import responseCodes.exception.BadRequestException;
+import responseCodes.exception.ForbiddenException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+
+import static constants.ControllerConstants.LOCATION;
+import static constants.ControllerConstants.OVER_RANGE;
+import static constants.ControllerConstants.WRONG_EXPRESSION;
+
+public class ServletController implements Controller {
+
+    private static final WebCalculatorFactory FACTORY = new WebCalculatorFactory();
+    private static CalculatorService calculatorService;
+
+    public void getResult(HttpServletRequest httpServletRequest, HttpServletResponse response) {
+        calculatorService = FACTORY.createCalculator();
+        String sessionID = getSessionId(httpServletRequest);
+
+        try {
+            int result = calculatorService.calculate(sessionID);
+            PrintWriter printWriter = response.getWriter();
+            printWriter.printf(String.valueOf(result));
+            response.setStatus(HttpServletResponse.SC_OK);
+            printWriter.close();
+        } catch (Exception e) {
```

Comment:
Please use more specific errors to indetify error code. For example:
![image](https://user-images.githubusercontent.com/20165935/136108100-2b89dba1-8726-4fe1-8bbe-d6edc1a4a6cb.png)